common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -177,6 +206,45 @@ def _check_provider_configurations(self):
 
         return metrics
 
+    def _check_no_config(self, provider_config, provider_info, disabled_config_count, null_config_count):
+        """Helper to check providers with no direct SAML configuration."""
+        try:
+            default_config = SAMLConfiguration.current(provider_config.site_id, 'default')
+            if not default_config or default_config.id is None:
+                # Resolution: Create a SAML configuration for this provider
+                # or create a default configuration for the site
+                self.stdout.write(
+                    f"[WARNING] {provider_info} has no direct SAML configuration and "
+                    "no matching default configuration was found."
+                )
+                null_config_count += 1
+                return disabled_config_count, null_config_count
+
+            if not default_config.enabled:
+                disabled_config_count += 1
```

Comment:
This seems like a null config, with just a slightly different issue around the default.
```suggestion
                null_config_count += 1
```